tests/callbacks/test_optimizer_monitor.py

```diff
@@ -71,29 +76,116 @@ def test_fsdp_optimizer_monitor(device, world_size):
                       optimizers=DecoupledAdamW(model.parameters()),
                       max_duration='3ba',
                       fsdp_config={
-                          'sharding_strategy': 'FULL_SHARD',
-                          'min_params': 10,
+                          'sharding_strategy': 'FULL_SHARD' if world_size > 1 else 'NO_SHARD',
+                          'min_params': 1,
```

Comment:
this is about to get removed iirc / is deprecated. Can be separate PR to fix? CC: @bcui19

Reply:
I think I can just remove it tbh, might need to set some `_fsdp_wrap` or something.